### PR TITLE
revamp Hardware Setup chapter

### PR DESCRIPTION
**Pull request by Be-ing**
Add section for using a splitter cable, divide controller section into subsections for controllers with and without soundcards, and many other language changes. This needs a diagram for a setup with a splitter cable.

### CODE REVIEW

**Comment by Be-ing** [comment]
I updated http://mixxx.org/wiki/doku.php/hardware_compatibility to mention splitter cables and list available models.

File: source/chapters/setup.rst [hunk 1]
```diff
@@ -4,36 +4,43 @@ Hardware Setup
...
    S.Brandt <s.brandt@mixxx.org>
+   Be <be.0@gmx.com>
 
 This chapter describes the most common hardware setups to serve as examples. We
-will go over the setup of timecode records/CDs, MIDI control and keyboard
-control.
-
-Audio Output
-============
+will go over the setup of soundcards, controllers, and timecode records/CDs.
```
Comment (kain88-de) [inline source/chapters/setup.rst]: This is not a complete sentence. Please add a subject

Comment (Be-ing) [inline source/chapters/setup.rst]: Yes it is. You may have missed the "We" at the beginning of the sentence because you were looking at the diff.

**Comment by Be-ing** [comment]
Could someone review this?

File: source/chapters/setup.rst [hunk 2]
```diff
@@ -4,36 +4,43 @@ Hardware Setup
...
+headphones. Being able to cue is a crucial aspect of DJing that separates DJing
+from merely making a playlist.
+
+In order to use headphone cueing, you will need at least 2 separate audio
+outputs, typically provided by a USB soundcard (also known as an “audio 
+interface”). The headphone jack on most laptops is not a second audio output. 
+Rather, plugging headphones into the jack simply redirects the laptop's single
+stereo output to your headphones. However, a splitter cable can be used to
+separate the stereo output of a headphone jack into two separate mono outputs
+for headphone cueing.
```
Comment (ywwg) [inline source/chapters/setup.rst]: "... for headphone cueing, but you will have a much better experience with a dedicated sound card that has at least two stereo outputs.  See [wiki link] for a list of recommended sound cards."

File: source/chapters/setup.rst [hunk 3]
```diff
@@ -4,36 +4,43 @@ Hardware Setup
...
+
+Unlike some proprietary DJ systems, Mixxx can use any soundcard and any
+:term:`MIDI<MIDI controller>` or :term:`HID` controller that your
+:term:`OS <operating system>` has drivers to use. Mixxx can use multiple audio
+devices simultaneously. Depending on your budget and application area, your
+sound setup and requirements may vary. This section provides useful information
+for club, hobby and radio DJs alike.
+
+.. seealso:: The `Mixxx DJ Hardware Guide
```
Comment (ywwg) [inline source/chapters/setup.rst]: oh I guess here's the link I mentioned :)

File: source/chapters/setup.rst [hunk 4]
```diff
@@ -66,13 +73,58 @@ out**.
...
+#. Check that your headphones are plugged into the side of the cable with a
+   headphones symbol and that the speakers are plugged into the side of the cable
+   with a speaker symbol.
+#. From the :guilabel:`Master` drop-down menu, select your built-in soundcard,
+   then :guilabel:`Channel 1`
+#. From the :guilabel:`Headphones` drop-down menu, select your built-in 
+   soundcard, then :guilabel:`Channel 2`
+#. Click :guilabel:`Apply` to save the changes.
+
+.. warning:: Most splitter cables, particularly those marketed as headphone
```
Comment (ywwg) [inline source/chapters/setup.rst]: most -> many

File: source/chapters/setup.rst [hunk 5]
```diff
@@ -85,86 +137,111 @@ Laptop and an External USB Soundcard
...
+#. From the :guilabel:`Master` drop-down menu, select the external soundcard,
    then :guilabel:`Channels 1-2`
-#. From the :guilabel:`Headphones` drop-down menus, select the external
-   soundcard, then :guilabel:`Channels 3-4`
+#. From the :guilabel:`Headphones` drop-down menu, select the built-in
+   soundcard, then :guilabel:`Channels 1-2`
 #. Click :guilabel:`Apply` to save the changes.
 
-**Using two soundcards (built-in + external)**
+**Using an external multi-channel soundcard**
```
Comment (ywwg) [inline source/chapters/setup.rst]: add a few sentences about the multi-card sync options we have.  "no delay" is pretty good these days, despite the "experimental" tag in the UI

File: source/chapters/setup.rst [hunk 6]
```diff
@@ -85,86 +137,111 @@ Laptop and an External USB Soundcard
...
-   soundcard, then :guilabel:`Channels 1-2`
+#. From the :guilabel:`Headphones` drop-down menu, select the external
+   soundcard, then :guilabel:`Channels 3-4`
 #. Click :guilabel:`Apply` to save the changes.
 
+.. hint:: If you have technical difficulties with your external sound card, a
+          :ref:`splitter cable<setup-laptop-with-splitter>` can be used as a
+          portable, inexpensive backup.
+
+.. note:: Soundcards designed and marketed for recording often have a knob that
```
Comment (ywwg) [inline source/chapters/setup.rst]: This note is a little hard to understand.  I think the bit about "latency" is not important to mention here.  How about:

"Soundcards designed and marketed for recording often have a knob that allows you to mix its input directly with its output.  This function can be helpful when recording, but the master output level will be low even if there is no sound on the inputs.  To get the maximum output volume from these soundcards, you may need to set this knob to only send output from the computer and none from the input(s)."

File: source/chapters/setup.rst [hunk 7]
```diff
@@ -85,86 +137,111 @@ Laptop and an External USB Soundcard
...
-Mixxx can work with any :term:`MIDI controller` that has drivers for your
-:term:`OS <operating system>`, as long as there is a MIDI mapping file to tell
-Mixxx how to understand it. Mixxx comes bundled with a number of MIDI mapping
-presets. To see the full list, see :ref:`control-midi`.
+Most DJs prefer the tactile and intuitive control provided by DJ MIDI 
+controllers over only a keyboard and mouse. Mixxx can work with any :term:`MIDI 
+controller` as long as there is a MIDI mapping file to tell Mixxx how to 
+understand the controller's signals. Mixxx comes bundled with a number of MIDI
+mapping presets. To see the full list of included mappings, see
+:ref:`control-midi`. For controllers that Mixxx does not yet support, anyone can
```
Comment (ywwg) [inline source/chapters/setup.rst]: "for controllers that mixxx does not yet support, check the forums to see if someone has created a mapping [link], or you can do the mapping yourself [link"

**Comment by ywwg** [comment]
Very good, just some minor tweaks and additions that shouldn't take too long to incorporate

File: source/chapters/setup.rst (hunk 4 above)
Comment (Be-ing) [inline source/chapters/setup.rst]: AFAIK, there are only two commercially available DJ splitter cables. See http://mixxx.org/wiki/doku.php/hardware_compatibility#splitter_cables

Come to think of it, I'll add that link to the manual.

File: source/chapters/setup.rst (hunk 5 above)
Comment (Be-ing) [inline source/chapters/setup.rst]: I don't really know the differences between the options. Do you? Could you add that in after merging this PR?

Comment (ywwg) [inline source/chapters/setup.rst]: sure, can you file an Issue so we don't forget?

File: source/chapters/setup.rst (hunk 2 above)
Comment (Be-ing) [inline source/chapters/setup.rst]: I think the diagrams should be changed to depict the recommended type of sound card. I'll open an issue for this.

**Comment by Be-ing** [comment]
Notes addressed and a few other changes made.

**Comment by ywwg** [comment]
There's just that one sentence I think we can safely do without, and then this is good to merge

**Comment by Be-ing** [comment]
Removed that sentence.

**Comment by Be-ing** [comment]
It would be good if someone with a Griffin DJ cable checked the splitter cable instructions. I suspect it is wired the same way as the Native Instruments splitter cable, with the end marked for the master output as channel 1 (tip) and the end marked for the headphone output as channel 2 (ring).

**Comment by DJChloe** [comment]
Unfortunately I can confirm only for the traktor cable. I do not owe the Griffin one.

**Pull request merged**